safehttp/response_writer.go

```diff
@@ -54,7 +54,11 @@ func (w *ResponseWriter) WriteTemplate(t Template, data interface{}) Result {
 }
 
 // ServerError TODO
-func (w *ResponseWriter) ServerError(code StatusCode, resp Response) Result {
+func (w *ResponseWriter) ServerError(code StatusCode) Result {
+	if code < 500 || code >= 600 {
```

Comment:
Should we return an empty safe Result here? The user of the framework basically called the wrong method.

Reply:
I agree we should somehow signal to the user that they used the wrong function. I guess the options are:
- Return an error
- Embed error in Result somehow
- panic
What do you think? @mihalimara22 @kele @empijei

Reply:
Unclear. Add a TODO for now.